tests/node/rpc/messages/SubscribeVehicleDataResponseTests.js

```diff
@@ -1,8 +1,14 @@
 const SDL = require('../../../config.js').node;
+
+// messages
 const SubscribeVehicleDataResponse = SDL.rpc.messages.SubscribeVehicleDataResponse;
+
+// enums
 const FunctionID = SDL.rpc.enums.FunctionID;
 const MessageType = SDL.rpc.enums.MessageType;
 const VehicleDataType = SDL.rpc.enums.VehicleDataType;
+
+// structs
 const VehicleDataResult = SDL.rpc.structs.VehicleDataResult;
 
```

Comment:
**Optional:** This is not related to this PR, but you could make these changes if you would like:
On line 26 you could change `JSON_VEHICLEDATARESULT` to `JSON_HANDSOFFSTEERING`.

Reply:
Based on previous experience, it is better to avoid changes not related to PR. Will add if PMs request this in the review.